.gitignore

```diff
@@ -3,9 +3,7 @@ _build
 _static 
 
 #Data 
-tutorial2/data/full_timeseries/asf_rtcs
-tutorial2/data/full_timeseries/intermediate_cubes/s1_pc_cube.zarr
-tutorial2/data/subset_timeseries/asf_rtcs
+**/data/*
```

Comment:
Actually, it looks like for this to work you have to update the git index since these zarrs are already tracked:
https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore